packages/launchpad/src/index.ts

```diff
@@ -29,6 +29,7 @@ export {
   isSearchByTagsQuery,
 } from "./cosmosclient";
 export { makeSignBytes } from "./encoding";
+export { buildFeeTable, FeeTable, GasLimits, GasPrice } from "./gas";
```

Comment:
Do we need to export `buildFeeTable`? My impression was this becomes a fully internal calculation with the new API

Reply:
I import it in CosmWasm. Should I be copying it over into that package instead?

Reply:
I see, good.